eland/tests/dataframe/test_dtypes_pytest.py

```diff
@@ -42,3 +43,7 @@ def test_flights_select_dtypes(self):
             pd_flights.select_dtypes(include=np.number),
             ed_flights.select_dtypes(include=np.number),
         )
+
+    def test_emtpy_series_dtypes(self):
+        s = build_series({})
```

Comment:
Let's also verify that no warnings are raised. Can use record=True on capture_warnings().